# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/filters/backdrop-filter-chain.png
# Commit: 79f4180c783b
# Full Hash: 79f4180c783b1e72fccb1e49fb8db086ea12ecca
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-05-18 09:47:25
# Regressor Bug: 1749625
# File Overlap Count: 1
# Description:
#   Bug 1749625 - Fix up and re-enable backdrop-filter r=gfx-reviewers,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D146643
# ==============================================================================

diff -r 622b05c84338 -r 79f4180c783b gfx/wr/wrench/reftests/filters/backdrop-filter-chain.png
Binary file gfx/wr/wrench/reftests/filters/backdrop-filter-chain.png has changed